src/Author.php

```diff
@@ -71,9 +80,15 @@ private function get_author_data( \WP_User $user ) {
 	public function endpoint_args() {
 		return [
 			'id' => [
-				'required' => true,
-				'sanitize_callback' => function ( $author_id, $request, $key ) {
-					return absint( $author_id );
+				'default' => false,
```

Comment:
It looks ok to me, we're saying it's valid if it's a positive int or doesn't exist